src/main/java/seedu/address/model/person/Organization.java

```diff
@@ -0,0 +1,109 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents an Organisation in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Organization extends Contact {
+
+    private final Status status;
+    private final Position position;
+    //private final Url url;
+
+    // Data fields
+    //private final Set<Recruiter> tags = new HashSet<>();
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Organization(Name name, Phone phone, Email email, Address address,
+                        Set<Tag> tags, Status status, Position position) {
+        super(name, phone, email, address, tags);
+        requireAllNonNull(status, position);
+        this.status = status;
+        this.position = position;
+    }
+
+    @Override
+    public Type getType() {
+        return Type.ORGANIZATION;
+    }
+
+    public Status getStatus() {
+        return status;
+    }
+
+    public Position getPosition() {
+        return position;
+    }
+
+    //    public Address getUrl() {
+    //        return url;
+    //    }
+
+    //    public Set<Recruiter> getTags() {
+    //        return Collections.unmodifiableSet(recruiters);
+    //    }
+
+    /**
+     * Returns true if both contacts have the same name.
+     * This defines a weaker notion of equality between two contacts.
+     */
+    public boolean isSameOrganization(Organization otherOrganization) {
+        if (otherOrganization == this) {
+            return true;
+        }
+
+        return otherOrganization != null
+                && otherOrganization.getName().equals(getName());
+    }
+
```

Comment:
I would suggest not having the `isSameOrganization` method since there's no immediate benefit. We can simply use the inherited `isSameContact` instead. 

Additionally, we should change `isSameContact` to use `id` for equality.